tools/bootnode/bootnode.go

```diff
@@ -44,6 +46,13 @@ func main() {
 
 	if *debug {
 		logrus.SetLevel(logrus.DebugLevel)
+
+		// Geth specific logging.
+		glogger := gethlog.NewGlogHandler(gethlog.StreamHandler(os.Stderr, gethlog.TerminalFormat(false)))
+		glogger.Verbosity(gethlog.LvlTrace)
+		gethlog.Root().SetHandler(glogger)
+
+		log.Debug("Debug logging enabled.")
```

Comment:
maybe it will be good to mention its go-ethereum debug logs

Reply:
Oh sorry, didn't see this. This should be OK as is.